web/components/bet-button.tsx

```diff
@@ -9,8 +9,8 @@ import { useUserContractBets } from 'web/hooks/use-user-bets'
 import { useSaveBinaryShares } from './use-save-binary-shares'
 import { Col } from './layout/col'
 
-// Inline version of a bet panel. Opens BetPanel in a new modal.
-export default function BetRow(props: {
+/** Button that opens BetPanel in a new modal */
+export default function BetButton(props: {
```

Comment:
I would switch this to not be a default export, since you're editing this. Thanks~

Also, thanks for renaming it to something more appropriate!